.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: php
+
+matrix:
+    include:
+        - php: 7.1
+        - php: 7.0
+        - php: 5.6
+        - php: 5.5
+          env: REMOVE_TWITEROAUTH=true
+        - php: 5.4
```

Comment:
Selfoss still supports PHP 5.4. As does [Debian Wheezy][1], which is on LTS until [May 2018][2].

[1]: https://wiki.debian.org/PHP#Available_versions
[2]: https://wiki.debian.org/LTS